ddtrace/bootstrap/sitecustomize.py

```diff
@@ -22,15 +22,20 @@
 
 from ddtrace.internal.telemetry import telemetry_writer
 from ddtrace import config  # noqa:F401
-from ddtrace._logger import _configure_log_injection
+from ddtrace._logger import DD_LOG_FORMAT
 from ddtrace.internal.logger import get_logger  # noqa:F401
 from ddtrace.internal.module import ModuleWatchdog  # noqa:F401
 from ddtrace.internal.module import is_module_installed
 from ddtrace.internal.utils.formats import asbool  # noqa:F401
 
 # Debug mode from the tracer will do the same here, so only need to do this otherwise.
 if config._logs_injection:
-    _configure_log_injection()
+    from ddtrace import patch
+
+    patch(logging=True)
+    ddtrace_logger = logging.getLogger("ddtrace")
+    for handler in ddtrace_logger.handlers:
+        handler.setFormatter(logging.Formatter(DD_LOG_FORMAT))
```

Comment:
Can we also register the log injection RC hook here. It would be nice if we could move it outside of tracer initialization: https://github.com/DataDog/dd-trace-py/blob/v3.3.0rc3/ddtrace/_trace/tracer.py#L1132-#L1140

Reply:
👍  yep we can do that in a follow-up